examples/clients/abe/api/swagger.yaml

```diff
@@ -2189,8 +2198,6 @@ definitions:
     properties:
       value:
         type: "string"
-    example:
-      value: "value"
```

Comment:
Do you know why this disappeared? It doesn't look particularly useful, but it would be interesting to understand.

Reply:
I do not know why this disappeared.  It appears that this file is generated via this command:

```
swagger-codegen generate -i examples/proto/examplepb/a_bit_of_everything.swagger.json \
    -l go -o examples/clients/abe --additional-properties packageName=abe
```

And I see no corresponding change to the `a_bit_of_everything.swagger.json` input file.

Perhaps the swagger-codegen tool is being "smart" and omitting the default example on the schema, because all usages of the schema in this file (just the one Echo grpc method) have a custom response example?

Reply:
Yeah, don't worry about it.